### ISSUE TITLE
Images are not getting added in tf.event files

### ISSUE THREAD

**Issue by kulkarnivishal** [tensorflow/models#5798]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:
models/research/object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
TPU
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
1.9
- **Bazel version (if compiling from source)**:
NA
- **CUDA/cuDNN version**:
NA (Cloud ML engine TPU)
- **GPU model and memory**:
TPU basic tier
- **Exact command to reproduce**:
Training: 
```gcloud ml-engine jobs submit training $JOB_NAME_`date +%m_%d_%Y_%H_%M_%S` --job-dir=${MODEL_DIR} --packages dist/object_detection-0.1.tar.gz,slim/dist/slim-0.1.tar.gz,/tmp/pycocotools/pycocotools-2.0.tar.gz --module-name object_detection.model_tpu_main --runtime-version 1.9 --scale-tier BASIC_TPU --region us-central1 -- \ --tpu_zone us-central1 --model_dir=${MODEL_DIR} --pipeline_config_path=${PIPELINE_CONFIG_PATH}```
Evaluation: 
```gcloud ml-engine jobs submit training $JOB_NAME_eval_`date +%m_%d_%Y_%H_%M_%S` --runtime-version 1.9 --job-dir=${MODEL_DIR}    --packages dist/object_detection-0.1.tar.gz,slim/dist/slim-0.1.tar.gz,/tmp/pycocotools/pycocotools-2.0.tar.gz --module-name object_detection.model_main --region us-central1 --scale-tier BASIC_GPU -- \ --model_dir=${MODEL_DIR} --pipeline_config_path=${PIPELINE_CONFIG_PATH} --checkpoint_dir=${MODEL_DIR}```

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.
**Images are not getting added to event files and because of this the tensorboard shows no information apart from loss and learning rate.** 

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.
![image](https://user-images.githubusercontent.com/36867201/48858825-3e78ed80-ed71-11e8-926e-f9eda3744879.png)

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
